document/docs/zh/config/options/features-rspack.md

```diff
@@ -0,0 +1,24 @@
+```ts
```

Comment:
The document fragments should be moved to an unified directory, such as `shared` or `fragments`.

And they should be excluded from `routes`, see: https://github.com/web-infra-dev/rspress/blob/main/packages/document/rspress.config.ts#L53-L55